### PR TITLE
Notebooks for mxnet detectors

### PR DESCRIPTION
**Pull request by vivinousi**
added four jupyter notebooks for mxnet-based detectors

### CODE REVIEW

File: projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+   ],
+   "source": [
+    "learner.load(\"./centernet_default\", verbose=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will also download one sample image and format it so we can use it in opendr for testing:"
```
Comment (passalis) [inline projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb]:
```suggestion
    "We will also download one sample image and load it so we can use it in OpenDR for testing:"
```

File: projects/perception/object_detection_2d/retinaface/inference_tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+   "source": [
+    "Note that you must change the loading direction if you change the backbone of the learner."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baking-alliance",
+   "metadata": {},
+   "source": [
+    "We will also download one sample image and format it so we can use it in opendr for testing:"
```
Comment (passalis) [inline projects/perception/object_detection_2d/retinaface/inference_tutorial.ipynb]:
```suggestion
    "We will also download one sample image and load it so we can use it in OpenDR for testing:"
```

File: projects/perception/object_detection_2d/ssd/inference_tutorial.ipynb [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+   "source": [
+    "learner.load(\"./ssd_default_person\", verbose=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baking-alliance",
+   "metadata": {},
+   "source": [
+    "We will also download one sample image and format it so we can use it in opendr for testing:"
```
Comment (passalis) [inline projects/perception/object_detection_2d/ssd/inference_tutorial.ipynb]:
```suggestion
    "We will also download one sample image and load it so we can use it in OpenDR for testing:"
```

File: projects/perception/object_detection_2d/yolov3/inference_tutorial.ipynb [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+   "source": [
+    "learner.load(\"./yolo_default\", verbose=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baking-alliance",
+   "metadata": {},
+   "source": [
+    "We will also download one sample image and format it so we can use it in opendr for testing:"
```
Comment (passalis) [inline projects/perception/object_detection_2d/yolov3/inference_tutorial.ipynb]:
```suggestion
    "We will also download one sample image and load it so we can use it in OpenDR for testing:"
```

**Review by passalis — APPROVED** [review]
Thank you!  Seems fine to me. I have proposed a few changes.

File: projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/administrator/opendr/venv/lib/python3.8/site-packages/gluoncv/__init__.py:40: UserWarning: Both `mxnet==1.8.0` and `torch==1.7.1` are installed. You might encounter increased GPU memory footprint if both framework are used at the same time.\n",
+      "  warnings.warn(f'Both `mxnet=={mx.__version__}` and `torch=={torch.__version__}` are installed. '\n"
+     ]
+    }
+   ],
+   "source": [
+    "from opendr.perception.object_detection_2d.centernet.centernet_learner import CenterNetDetectorLearner\n",
```
Comment (ad-daniel) [inline projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb]:
```suggestion
    "from opendr.perception.object_detection_2d import CenterNetDetectorLearner\n",
```

Comment (ad-daniel) [reply projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb]: import can be shortened

File: projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb (hunk 1 above)
Comment (ad-daniel) [inline projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb]:
```suggestion
    "We will also download one sample image and format it so we can use it in OpenDR for testing:"
```

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Thank you, worked all four for me. Only two small comments

File: projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb (hunk 1 above)
Comment (vivinousi) [reply projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb]: Fixed with [d97cd1b](https://github.com/opendr-eu/opendr/pull/230/commits/d97cd1b42f9fdd581b36cb3bddc07e94dde38fdf), for some reason I could not automatically commit suggestions

**Review by ad-daniel — APPROVED** [review]
Thank you!

**Review by passalis — APPROVED** [review]
Thank you!

**Pull request merged**